src/PIL/ImageFont.py

```diff
@@ -804,7 +808,8 @@ def truetype(
 ) -> FreeTypeFont:
     """
     Load a TrueType or OpenType font from a file or file-like object,
-    and create a font object.
+    and create a font object. For loading bitmap fonts instead,
+    see :py:func:`~PIL.ImageFont.load` and :py:func:`~PIL.ImageFont.load_path`.
     This function loads a font object from the given file or file-like
     object, and creates a font object for a font of the given size.
```

Comment:
```suggestion
    and create a font object. This function loads a font object from the given
    file or file-like object, and creates a font object for a font of the given
    size. For loading bitmap fonts instead, see :py:func:`~PIL.ImageFont.load`
    and :py:func:`~PIL.ImageFont.load_path`.
```
Very hopefully my last nitpick - I think the links should be at the end of the paragraph, like in `load()`, so this swaps the sentences.